tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -179,6 +182,25 @@ def on_heartbeat(self, cluster_data):
         # Sync the OSD utilization details
         self._sync_osd_utilization()
 
+    def _sync_cluster_network_details(self):
+        try:
+            NS.tendrl.objects.ClusterNetworkDetails(
+                integration_id=NS.tendrl_context.integration_id
+            ).load()
+        except etcd.EtcdKeyNotFound:
+            try:
+                cluster_config = NS.ceph.objects.SyncObject(
+                    sync_type='config'
+                ).load().data
+                NS.tendrl.objects.ClusterNetworkDetails(
```

Comment:
ClusterNetworkDetails object should be `Cluster` 

check my comments here https://github.com/Tendrl/commons/pull/631